bundles/org.openhab.binding.millheat/NOTICE

```diff
@@ -11,3 +11,5 @@ https://www.eclipse.org/legal/epl-2.0/.
 == Source Code
 
 https://github.com/openhab/openhab2-addons
+
+Contains code from https://stackoverflow.com/questions/50318736/how-to-log-httpclient-requests-response-including-body
```

Comment:
stackoverflow code is MIT. You don't need to reference it here, if it's not a 1:1 copy in it's entirety. Believe me all programmers get solutions and ideas from stackoverflow, the NOTICE files would be crammed if we reference to every helpful post.

Reply:
I agree. Could you clarify this comment? https://github.com/openhab/openhab2-addons/pull/5077#issuecomment-485889867

Reply:
The important part of my statement above is "**if it's not a 1:1 copy in it's entirety**". If it is a copy, the author must be credited, which goes hand in hand with my other statement in the referenced post ^^.

Reply:
Also this one https://github.com/openhab/openhab2-addons/pull/5077#issuecomment-490625449

Reply:
If in doubt, the developer documentation is the source of truth ^^. 
I took the liberty and added a few lines: https://github.com/openhab/openhab-docs/pull/977/files#diff-1a7464c896e1a289d3025203ed376370R184

Reply:
Removed